### PR TITLE
dev-ml/stringext: new ebuild

### PR DESCRIPTION
**Pull request by tomboy-64**
- needed as a transitive dependency of Libre S3 1.1 (net-misc/libres3)

@aballier

### CODE REVIEW

File: dev-ml/stringext/stringext-1.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="
+	test? ( dev-ml/qcheck[ounit(-)] )
+	>=dev-lang/ocaml-4
+"
+RDEPEND=""
```
Comment (mgorny) [inline dev-ml/stringext/stringext-1.4.0.ebuild]: Hmm, doesn't it require ocaml-4 at runtime as well?

**Comment by idella** [comment]
@gentoo/proxy-maint

File: dev-ml/stringext/stringext-1.4.0.ebuild (hunk 1 above)
Comment (aballier) [inline dev-ml/stringext/stringext-1.4.0.ebuild]: probably; oasis.eclass adds a := dep on ocaml, meaning that if ocaml is downgraded it needs to be rebuilt, so this dep effectively makes it a RDEPEND on ocaml-4 actually, hence better move it to RDEPEND (with a := just for clarity)

**Comment by aballier** [comment]
please move ocaml to RDEPEND and it's good to go

**Comment by tomboy-64** [comment]
@aballier Done. Good to go.

**Comment by mgorny** [comment]
@tomboy-64, I think he meant having it both in R- and DEPEND ;-).

**Comment by aballier** [comment]
yes it should be both :)

**Comment by tomboy-64** [comment]
that's what you get for doing gentoo stuff shortly after waking up.

sorry. done.

@aballier

**Comment by aballier** [comment]
merged, thx!

**Pull request closed without merging**